jcommune-model/src/test/java/org/jtalks/jcommune/model/dao/hibernate/TopicDraftHibernateDaoTest.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.jtalks.jcommune.model.dao.hibernate;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.classic.Session;
+import org.jtalks.jcommune.model.dao.TopicDraftDao;
+import org.jtalks.jcommune.model.entity.JCUser;
+import org.jtalks.jcommune.model.entity.ObjectsFactory;
+import org.jtalks.jcommune.model.entity.PersistedObjectsFactory;
+import org.jtalks.jcommune.model.entity.TopicDraft;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTransactionalTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionConfiguration;
+import org.springframework.transaction.annotation.Transactional;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author Dmitry S. Dolzhenko
+ */
+@ContextConfiguration(locations = {"classpath:/org/jtalks/jcommune/model/entity/applicationContext-dao.xml"})
+@TransactionConfiguration(transactionManager = "transactionManager", defaultRollback = true)
+@Transactional
+public class TopicDraftHibernateDaoTest extends AbstractTransactionalTestNGSpringContextTests {
```

Comment:
We started to write Component Tests in Groovy. If you like, we could start writing DB and Unit tests in Groovy as well.